<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>900</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSetWorkspace"/>
    <addaction name="separator"/>
    <addaction name="actionReloadParameters"/>
    <addaction name="separator"/>
    <addaction name="actionSaveBBox"/>
    <addaction name="actionLoadBBox"/>
    <addaction name="actionSaveImage"/>
   </widget>
   <widget class="QMenu" name="menuVisualization">
    <property name="title">
     <string>Visualization</string>
    </property>
    <addaction name="actionToggleTrajPoints"/>
    <addaction name="actionToggleMapImage"/>
    <addaction name="actionSwitchViewMode"/>
   </widget>
   <widget class="QMenu" name="menuAlgorithm">
    <property name="title">
     <string>Algorithm</string>
    </property>
    <addaction name="actionVoteMapImage"/>
    <addaction name="actionInitializeSkeleton"/>
    <addaction name="actionExtractSkeleton"/>
    <addaction name="actionExtractSkeletonStep"/>
    <addaction name="actionComputeSkeletonGraph"/>
    <addaction name="actionClusterJunctionTrajectory"/>
    <addaction name="actionSaveSelectedClusters"/>
    <addaction name="actionCompleteTrajectories"/>
    <addaction name="actionFilterJunctionEdge"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuVisualization"/>
   <addaction name="menuAlgorithm"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar_1</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionToggleTrajPoints"/>
   <addaction name="actionToggleMapImage"/>
   <addaction name="actionSwitchViewMode"/>
   <addaction name="actionVoteMapImage"/>
   <addaction name="actionInitializeSkeleton"/>
   <addaction name="actionExtractSkeleton"/>
   <addaction name="actionExtractSkeletonStep"/>
   <addaction name="actionIncrementH0"/>
   <addaction name="actionComputeSkeletonGraph"/>
       <addaction name="actionFilterJunctionEdge"/>
   <addaction name="actionClusterJunctionTrajectory"/>
   <addaction name="actionCompleteTrajectories"/>
  </widget>
  <widget class="QDockWidget" name="dockWidgetParameterEditor">
   <property name="allowedAreas">
    <set>Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Parameter Editor</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPlainTextEdit" name="plainTextEditParameterEditor"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonUpdateParameters">
       <property name="text">
        <string>Update Parameters</string>
       </property>
       <property name="shortcut">
        <string>Ctrl+S</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockOptions">
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Option</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockOptionContents">
    <layout class="QVBoxLayout" name="verticalLayout1">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <!--
	 <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
	 <item>
	   <widget class="QLabel" name="minFlowSizeLabel">
	     <property name="text">
	       <string>Minimum flow size</string>
	     </property>
	   </widget>
	   
	 </item>
	 <item>
	   <widget class="QDoubleSpinBox" name="minFlowSize"/>
	   <property name="maximum">
             <number>500</number>
           </property>
	 </item>
	 <item>
	   <widget class="QPushButton" name="buttonFilterEdge">
	     <property name="text">
	       <string>Apply</string>
	     </property>
	   </widget>
	 </item>
	 </layout>
	 
     </item>
-->
     <item>
      <widget class="QLabel" name="clusterSelectorLabel">
       <property name="text">
        <string>Select sub-trajectory cluster</string>
       </property>
       <property name="buddy">
        <cstring>clusterSelectorLabel</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="clusterSelector"/>
     </item>
     <item>
      <widget class="QPushButton" name="buttonSaveCluster">
       <property name="text">
        <string>Export Clusters to File</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="clusterFilterLabel">
       <property name="text">
        <string>Display clusters of minimum size</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QSlider" name="sliderClusterSize">
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>10</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksBelow</enum>
         </property>
         <property name="tickInterval">
          <number>4</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="buttonShowAllClusters">
         <property name="text">
          <string>Show All Clusters</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonHideAllClusters">
         <property name="text">
          <string>Hide All Clusters</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxTrajHeading">
         <property name="maximumSize">
          <size>
           <width>85</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Arrow</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="completeLabel">
         <property name="text">
          <string>Trajectory Id</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="trajIdSpinBox">
         <property name="maximum">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonShowDenseTraj">
         <property name="text">
          <string>Show Dense Trajectory</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="sigmaLabel">
       <property name="text">
        <string>Sigma at selected point: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonDisplaySigma">
       <property name="text">
        <string>Visualize Sigma</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonDisplayDensity">
       <property name="text">
        <string>Visualize Density</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionSetWorkspace">
   <property name="text">
    <string>Set Workspace</string>
   </property>
  </action>
  <action name="actionReloadParameters">
   <property name="text">
    <string>Load Parameters</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSwitchViewMode">
   <property name="text">
    <string>Switch View Mode</string>
   </property>
   <property name="shortcut">
    <string>V</string>
   </property>
  </action>
  <action name="actionSaveBBox">
   <property name="text">
    <string>Save All Bounding Boxes</string>
   </property>
  </action>
  <action name="actionLoadBBox">
   <property name="text">
    <string>Load Bounding Box</string>
   </property>
  </action>
  <action name="actionSaveImage">
    <property name="text">
      <string>Save Image As</string>
    </property>
  </action>
  <action name="actionVoteMapImage">
   <property name="text">
    <string>Vote Map Image</string>
   </property>
  </action>
  <action name="actionInitializeSkeleton">
   <property name="text">
    <string>Initialize L1 Medial Skeleton</string>
   </property>
  </action>
  <action name="actionExtractSkeleton">
   <property name="text">
    <string>Extract Skeleton</string>
   </property>
  </action>
  <action name="actionExtractSkeletonStep">
   <property name="text">
    <string>&gt;</string>
   </property>
  </action>
  <action name="actionIncrementH0">
   <property name="text">
    <string>+</string>
   </property>
  </action>
  <action name="actionComputeSkeletonGraph">
   <property name="text">
    <string>Compute Graph</string>
   </property>
  </action>
  <action name="actionCompleteTrajectories">
   <property name="text">
    <string>Complete Trajectories</string>
   </property>
  </action>
  <action name="actionClusterJunctionTrajectory">
   <property name="text">
    <string>Cluster junction trajectories</string>
   </property>
  </action>
  <action name="actionToggleTrajPoints">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Toggle Traj Points</string>
   </property>
  </action>
  <action name="actionToggleMapImage">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Toggle Map Image</string>
   </property>
  </action>
  <action name="actionSaveSelectedClusters">
   <property name="text">
    <string> Save Selected Clusters</string>
   </property>
   <property name="toolTip">
    <string>Save selected clusters to file</string>
   </property>
  </action>
  <action name="actionFilterJunctionEdge">
    <property name="text">
      <string>Filter junction edge by flow size</string>
    </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
